lib/main.js

```diff
@@ -175,6 +178,8 @@ module.exports = class Router extends EventEmitter {
     postBundleHandler() {
         return async (req, res, next) => {
             const { id, type } = req.params;
+            const sourceMaps = req.query.sourceMaps || false;
+            const minify = req.query.minify || isProduction;
```

Comment:
It would be great to coerce both of these to booleans beforehand

Reply:
also a logical error here - `req.query.minify === 'false'` for instance will be ignored